### ISSUE TITLE
Add GitLab support

### ISSUE THREAD

**Issue by NiranjanTambe** [aslushnikov/latex-online#70]
The program currently only supports GitHub, which is a non-free (free as in free speech) tool. Many developers find GitLab a better and a safer place to release their codes. It would be really great if this tool also supports GitLab along with GitHub. Currently if we paste a GitLab url in the tool, it generates an error saying `Not a GitHub repository`.

**Comment by aslushnikov** [comment]
Fair enough. Can you share your project link on GitLab for me to test with?

**Comment by NiranjanTambe** [comment]
Yes; sure. Here you go.

https://gitlab.com/niruvt/latexonline.cc/-/blob/main/test.tex
